README.md

```diff
@@ -1 +1,346 @@
-#  Chart repo
+# OpenShift Helm Charts Repository
+
+OpenShift Helm Charts Repository is the canonical source of all the certified
+Helm charts from [partners][partners] and the community. Once your chart is
+ready for submission, you can submit it as a pull request. The chart is going to
+be certified after running the [chart-verifier][chart-verifier] tool against
+it. You may also consider running the chart-verifier locally and submit the
+generated report for certification.
```

Comment:
Struggling to word this well but need to be clear in the full range of options:
 
Once your chart is ready for for submission, you can submit a pull request. The pull request can include the chart and/or a report generated by the [chart-verifier][chart-verifier] tool. Based on submission the content the chart will be certified and the chart-verifier will be run if a report is not provided.